packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -223,7 +223,7 @@ export default function(api, options) {
           prop.remove();
         }
 
-        if (!nodes.length) return;
+        if (!computedNodes.length && !afterNodes.length) return;
```

Comment:
Just a small nit, I would prefer to explicity check for: `computedNodes.length === 0 && afterNodes.length === 0`. It's more human-readable to me and also the engine don't need to coerce it to a boolean.

Reply:
sure, gonna change it in a moment and merge in

Reply:
Cool thanks